contracts/test/eip155.ts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { EIP155Tests__factory } from '../typechain-types/factories/contracts/tests';
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
+
+describe('EIP-155', function () {
+  async function deploy() {
+    const factory = (await ethers.getContractFactory(
+      'EIP155Tests',
+    )) as EIP155Tests__factory;
+    const x = await factory.deploy({ value: ethers.utils.parseEther('1') });
+    return { x };
```

Comment:
```suggestion
    return { testContract };
```
also, this appears to be a `beforeAll`, which I'd recommend using